docs/application/toc_all.md

```diff
@@ -158,6 +158,7 @@
 ##### [Package Manager](/application/dotnet/guides/app-management/package-manager.md)
 ##### [Message Port](/application/dotnet/guides/app-management/message-port.md)
 ##### [Data Control](/application/dotnet/guides/app-management/data-control.md)
+##### [Event Manager](/application/dotnet/guides/app-management/event.md)
```

Comment:
It looks like this "Event Manager" page is placed under "Data Control" according to this update.
However according to the fix of /app-management/overview.md, "Event Manager" is placed above "Package Manager".
Which one is correct?

Reply:
I've fixed the same as Native API

Reply:
@kilig Would you please match the order of the list between dotnet/guides/app-management/overview.md and toc_all.md? 
The toc_all.md list will be shown as the left navigation bar, and I think getting same order between the LNB and overview.md will be better.

In app-management/overview.md, the list order is 
 - ...
 - Package Manager
 - **Event Manager**
 - Message Port
 - ...

in toc_all.md, the list order is
 - ...
 - Application Manager
 - **Event Manager**
 - Package Manager
 - Message Port
 - ...

@divyk When @kilig updates this, would you please re-check and approve this PR again?

Reply:
Sure @mijin85cho , I will do it. Thanks.